nginx-error-pages/.gitlab-ci.yml

```diff
@@ -0,0 +1,31 @@
+Lint nginx-error-pages:
+  stage: "Quality"
+  image: hadolint/hadolint:v1.17.5-debian
+  script:
+    - hadolint ./nginx-error-pages/Dockerfile
+
+Build nginx-error-pages:
+  stage: "Build"
+  needs:
+    - Lint nginx-error-pages
+  extends: .base_register_to_gitlab_stage
+  variables:
+    CONTEXT: nginx-error-pages
+    IMAGE_NAME: ${CI_REGISTRY_IMAGE}/nginx-error-pages
+
+Test nginx-error-pages:
+  stage: "Test"
+  needs:
+    - Lint nginx-error-pages
+    - Build nginx-error-pages
+  image: "${CI_REGISTRY_IMAGE}/nginx-error-pages:${CI_COMMIT_SHA}"
+  script:
+    - /custom-error-pages
```

Comment:
You might want to look at 

https://github.com/SocialGouv/docker/blob/a8959f9/nginx4spa/.gitlab-ci.yml#L16-L45

To actually test the pages